web-ui/src/main/resources/catalog/components/edit/bounding/BoundingDirective.js

```diff
@@ -235,17 +235,21 @@
 
               // print output (skip if readonly)
               if (!ctrl.readOnly) {
+                // GML 3.2.1 is used for ISO19139:2007
+                // TODO: ISO19115-3:2018
                 ctrl.outputPolygonXml =
-                    '<polygon xmlns="http://www.isotc211.org/2005/gmd">' +
+                    '<gmd:polygon xmlns:gmd="http://www.isotc211.org/2005/gmd">' +
                     gnGeometryService.printGeometryOutput(
                     ctrl.map,
                     feature,
                     {
                       crs: outputCrs,
                       format: 'gml'
                     }
-                    ) +
-                    '</polygon>';
+                    ).replace(
+                      /http:\/\/www.opengis.net\/gml"/g,
```

Comment:
I think this is due to the OL version used in GeoNetwork doesn't support GML 3.2.0? If so would be good to add a comment in the code, so if it's updated to a version that supports it, this code can be removed.